h/templates/activity/search.html.jinja2

```diff
@@ -264,7 +280,7 @@
           </dt>
           <dd class="search-result-sidebar__dd">
             <a href="{{ user.uri }}" class="search-result-sidebar__user-link">
-              {{ user.domain }}
+              {{ pretty_link(user.uri) }}
```

Comment:
I think `user.uri` may sometimes be `""` or `None` or just a non-URL string. Does `pretty_link()` deal with these okay?